app/code/Magento/Contact/Controller/Index/Post.php

```diff
@@ -101,4 +65,61 @@ private function getDataPersistor()
 
         return $this->dataPersistor;
     }
+
+    /**
+     * @param array $post Post data from contact form
+     * @return void
+     */
+    protected function sendEmail($post)
+    {
+        $storeScope = \Magento\Store\Model\ScopeInterface::SCOPE_STORE;
+        $transport = $this->_transportBuilder
+            ->setTemplateIdentifier($this->scopeConfig->getValue(self::XML_PATH_EMAIL_TEMPLATE, $storeScope))
+            ->setTemplateOptions(
+                [
+                    'area' => \Magento\Backend\App\Area\FrontNameResolver::AREA_CODE,
+                    'store' => \Magento\Store\Model\Store::DEFAULT_STORE_ID,
```

Comment:
our static test failing with this message:
```
Module Magento\Contact has undeclared dependencies: hard [Magento\Backend]
```
do we really need the dependency on Backend here? If yes, please add the dependency to the composer.json otherwise.